src/pages/PhoneVerificationPage/PhoneVerificationOTPPage.tsx

```
@@ -0,0 +1,151 @@
+
+                <div className="text-center">
+                  {timer > 0 ? (
+                    <p className="text-sm text-[#828282]">
+                      Resend code in {timer}s
+                    </p>
+                  ) : (
+                    <button
+                      onClick={handleResend}
+                      disabled={resendMutation.isPending}
+                      className="text-sm font-medium text-[#2563eb] hover:underline disabled:opacity-50"
+                    >
```

Biome (2.3.13)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)